package.json

```diff
@@ -51,6 +51,7 @@
   "devDependencies": {
     "@types/node": "^6.0.45",
     "babel-core": "^6.16.0",
+    "browserify-versionify": "^1.0.6",
```

Comment:
I think we want this (or whatever new deps we add) in `lighthouse-extension/package.json` so that it's scoped to only install for those who are working on/with the extension